### PR TITLE
Test useTbTable

### PR DESCRIPTION
**Pull request by ErickGarciaRios**
Test useTbTable.ts

### CODE REVIEW

File: test/components/tbTableComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                            <tr key={index}>
+                                {state.columns
+                                    .filter(col => col.visible)
+                                    .map(col => {
+                                        return (
+                                            <td role="cell" key={col.name}>
+                                                {row[col.name]}
+                                            </td>
+                                        );
+                                    })}
```
Comment (geoperez) [inline test/components/tbTableComponent.tsx:35]: You don't need to use return in this lambda.

```suggestion
                                    .map(col => (
                                            <td role="cell" key={col.name}>
                                                {row[col.name]}
                                            </td>
                                        ))}
```

Comment (ErickGarciaRios) [reply test/components/tbTableComponent.tsx:35]: Removing...

File: test/components/tbTableComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <div>
+                <button onClick={() => api.sortColumn('CustomerName')}>Sort by Customer Name</button>
+            </div>
+            <table>
+                <thead>
+                    <tr role="rowheader">
+                        {state.columns
+                            .filter(col => col.visible)
+                            .map(col => {
+                                return <th key={col.name}>{col.label}</th>;
```
Comment (geoperez) [inline test/components/tbTableComponent.tsx:19]: And this little guy?

Comment (ErickGarciaRios) [reply test/components/tbTableComponent.tsx:19]: done

File: test/useTbTable.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    it('Should change setMultiSort value when pressing keyDown and keyUp', async () => {
+        const { container } = render(<TbTableComponent />);
+
+        fireEvent.keyDown(container, { key: 'Control', code: 'Control' });
+        fireEvent.keyUp(container, { key: 'Control', code: 'Control' });
+
+        expect(container).toBeDefined();
+    });
+
+    it('Pressing a key different to Control', async () => {
```
Comment (kadosh) [inline test/useTbTable.spec.tsx:31]: Please use a better description for this test. "Pressing a key different to Control" doesn't really express what you're testing.

**Pull request merged**